core/src/main/java/org/apache/iceberg/util/ThreadPools.java

```diff
@@ -59,6 +65,20 @@ public static ExecutorService getWorkerPool() {
     return WORKER_POOL;
   }
 
+  /**
+   * Return an {@link ExecutorService} for loading deletes.
+   *
+   * <p>The size of this worker pool limits the number of tasks concurrently reading delete files
+   * within a single JVM. In most cases, deletes are loaded whenever reading data on executors. The
+   * size of this pool has to be big enough to handle all data tasks running on the same executor.
+   *
+   * @return an {@link ExecutorService} that uses the delete worker pool
+   * @see SystemConfigs#DELETE_WORKER_THREAD_POOL_SIZE
+   */
+  public static ExecutorService getDeleteWorkerPool() {
```

Comment:
We don't usually use `get` but we have `getWorkerPool()` above so I matched it for consistency.